.github/ISSUE_TEMPLATE/shutdown_form.yaml

```diff
@@ -0,0 +1,50 @@
+name: Skip auto shutdown request
+description: Skip auto shutdown request
+body:
+
+- type: markdown
```

Comment:
i would make this a text field too, people will just leave this blank most likely